geomstats/riemannian_metric.py

```diff
@@ -215,7 +192,7 @@ def variance(self, points, weights=None, base_point=None):
         return variance
 
     def mean(self, points,
-             weights=None, n_max_iterations=100, epsilon=EPSILON):
```

Comment:
2 is a bit harsh no? Maybe 32 or 64 and call with n_max_iterations when called in the test?

Reply:
Yes, I just wanted to tests to run quickly. I'll change.